packages/core/src/carousel/private/types/CarouselRootProps.ts

```diff
@@ -4,6 +4,7 @@ type CarouselRootProps = {
   step?: number;
   selectedIndex?: number;
   autoSlideDelay?: number;
+  isSwipeDragDisabled?: boolean;
```

Comment:
Potential breaking change. Mentioned in AutoSlide example

Reply:
Updated.